libatalk/iniparser/iniparser.c

```diff
@@ -29,18 +29,16 @@ DEALINGS IN THE SOFTWARE.
 */
 
 /*---------------------------- Includes ------------------------------------*/
-#ifdef HAVE_CONFIG_H
 #include "config.h"
-#endif /* HAVE_CONFIG_H */
```

Comment:
Same as above. I do think it's safe to remove this, but it should be done so consistently across the codebase in one go, IMHO.